portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,8 +14,14 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
```

Comment:
How come you haven't made use of gson here?

Reply:
I realised that I forgot change to the gson method and was using my own. I managed to fix that. However, I made a separate branch (fix-0) as I've made a lot of changes from this branch.